rules/community/cloudtrail/cloudtrail_aws_access_by_evil.py

```diff
@@ -0,0 +1,26 @@
+"""Alert on matching IP address from aws access."""
+from stream_alert.rule_processor.rules_engine import StreamRules
+from helpers.base import fetch_values_by_datatype
+
+rule = StreamRules.rule
+
+
+@rule(logs=['cloudwatch:events'],
+      matchers=[],
+      datatypes=['ipv4'],
+      outputs=['aws-s3:sample-bucket',
+               'pagerduty:sample-integration',
+               'slack:sample-channel'])
+def cloudtrail_aws_access_by_evil(rec):
```

Comment:
We no longer put "sample" rules in the public repository, only real ones. I'll work with you offline to choose which ones to open source :)